packages/react-native-reanimated/Common/cpp/reanimated/NativeModules/ReanimatedModuleProxy.cpp

```diff
@@ -351,16 +351,17 @@ jsi::Value ReanimatedModuleProxy::getViewProp(
   const auto funPtr = std::make_shared<jsi::Function>(
       callback.getObject(rnRuntime).asFunction(rnRuntime));
   const auto shadowNode = shadowNodeFromValue(rnRuntime, shadowNodeWrapper);
-  uiScheduler_->scheduleOnUI([=]() {
+  uiScheduler_->scheduleOnUI(COPY_CAPTURE_WITH_THIS {
     jsi::Runtime &uiRuntime = uiWorkletRuntime_->getJSIRuntime();
     const auto resultStr =
         obtainPropFromShadowNode(uiRuntime, propNameStr, shadowNode);
 
-    jsScheduler_->scheduleOnJS([=](jsi::Runtime &rnRuntime) {
-      const auto resultValue =
-          jsi::String::createFromUtf8(rnRuntime, resultStr);
-      funPtr->call(rnRuntime, resultValue);
-    });
+    workletsModuleProxy_->getJSScheduler()->scheduleOnJS(
```

Comment:
Why can't we just store `jsScheduler_` and use it here? We use `uiScheduler_` anyways so let's be consistent.
```suggestion
    jsScheduler_->scheduleOnJS(
```

Reply:
I'm moving `UIScheduler` in https://github.com/software-mansion/react-native-reanimated/pull/6789 as well.

I removed the field in anticipation of an API coming. I don't think that with our idea of how Worklets&Reanimated should cooperate the ownership of schedulers should be shared.